src/boot/stylesheets/main.scss

```diff
@@ -690,14 +690,53 @@ body {
     				display: block;
     			}
     		}
+
+			pre {
+				background: $gray-light;
+				border: 1px solid lighten( $gray, 20% );
+				border-radius: 3px;
+				padding: 4px;
+
+				code {
+					border: none;
+					background: none;
+				}
+			}
```

Comment:
@drw158 I added this section here because I was getting funny styling with `<code><pre>`

<img width="189" alt="screen shot 2017-05-31 at 9 58 42 pm" src="https://cloud.githubusercontent.com/assets/5431237/26663446/495689d4-4650-11e7-8ef3-c987463c337f.png">

I would be very grateful if you reviewed this here CSS…